src/traces/table/plot.js

```diff
@@ -57,13 +57,14 @@ module.exports = function plot(gd, wrappedTraceHolders) {
         .classed(c.cn.tableControlView, true)
         .style('box-sizing', 'content-box');
     if(dynamic) {
+        let wheelEvent = 'onwheel' in document ? 'wheel' : 'mousewheel'
```

Comment:
Thanks for the PR.
To follow ES5 syntax, please use `var` instead of `let` here.